### ISSUE TITLE
Add support for MFA

### ISSUE THREAD

**Issue by bachya** [sissbruecker/linkding#252]
It would be great if the built-in authentication supported multi-factor authentication (ideally, through the TOTP standard). The [`django_otp`](https://django-otp-official.readthedocs.io/en/stable/overview.html) package might be helpful here.

(This would be an alternative to https://github.com/sissbruecker/linkding/issues/177.)

**Comment by clach04** [comment]
Alternative library https://github.com/jazzband/django-two-factor-auth

**Comment by bachya** [comment]
Good call, @clach04 – looks like that library [already includes `django_otp`](https://github.com/jazzband/django-two-factor-auth/blob/dae3fd143ed540a723125be3a3b87101082bd657/setup.py#L16). 👍🏻

**Comment by Mebus** [comment]
+1 

:-)

**Comment by esturniolo** [comment]
+1

**Comment by hwcltjn** [comment]
+1

**Comment by Mike710Shine** [comment]
+1

**Comment by Drallas** [comment]
A plus one from me too!

**Comment by sourcecodemage** [comment]
While passkey is probably better as it is slowly replacing TOTP, this also gets my vote!

**Comment by sokai** [comment]
I'll second this and give a quick _hint_, that https://github.com/pyauth/pyotp _may/would be_ IMHO well be suited for Linkding … KR

**Comment by ramsnerm** [comment]
+1
